README.md

```diff
@@ -218,51 +218,19 @@ npm run build-api-docs # TODO: this fails
 
 ### Running Locally
 
-The easiest way to run the API server locally is to use an Elasticsearch container running in Docker and `serverless-offline`.
-
-There is a `docker-compose.yml` file to simplify running Elasticsearch locally:
+Before the API can be run, Elasticsearch and Localstack need to be running. There is a `docker-compose.yml` file to simplify running Elasticsearch locally:
 
 ```sh
 docker-compose up -d
 ```
 
-The API can be run using `serverless-offline`, which will require a `serverless.yml` file. Copy `serverless.yml.example` to `serverless.yml`. You'll also need to edit the `serverless.yml` file to set the `ES_HOST` value.
-
-To do this, first change the value of `ES_HOST` in the serverless.yml file from the default (which dynamically populates it from the resource created upon deploy):
-
-```yml
-ES_HOST:
-  Fn::GetAtt: [ElasticSearchInstance, DomainEndpoint]
-```
-
-to instead use a hard-coded string of the Elasticsearch instance URL, e.g.,
+The API can then be run with:
```

Comment:
Where do I set ES_HOST if I want to run a local server against a remote ES instance?

Reply:
```sh
ES_HOST="http://remost.host" npm run serve
```